### PR TITLE
fix: clean snapshot artifacts on cleanApp hook

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Merge after https://github.com/NativeScript/nativescript-cli/pull/3339

This fixes NativeScript/nativescript-cli#3335

Ping @sis0k0 @rosen-vladimirov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-dev-webpack",
-  "version": "0.9.1",
+  "version": "0.10.0",
```
Comment (sis0k0) [inline package.json]: we bump the version right before releasing the plugin

**Comment by Mitko-Kerezov** [comment]
Ping @sis0k0 with fixed comment

**Comment by SvetoslavTsenov** [comment]
test sdk

**Comment by SvetoslavTsenov** [comment]
test uitest

**Comment by SvetoslavTsenov** [comment]
test uitest

**Comment by SvetoslavTsenov** [comment]
test uitest

**Comment by SvetoslavTsenov** [comment]
test uitests

**Comment by SvetoslavTsenov** [comment]
test build_uitests tns-core-modules#next

**Comment by SvetoslavTsenov** [comment]
test build_uitests tns_core_modules#next

**Comment by SvetoslavTsenov** [comment]
test build_uitests build_tns_core_modules#master

**Comment by SvetoslavTsenov** [comment]
test build_uitests build_tns_core_modules#master

**Comment by SvetoslavTsenov** [comment]
test build_uitests build_tns_core_modules#master

**Comment by SvetoslavTsenov** [comment]
test build_uitests build_tns_core_modules#master

**Comment by SvetoslavTsenov** [comment]
test build_uitests build_tns_core_modules#master

**Comment by SvetoslavTsenov** [comment]
test build_uitests build_tns_core_modules#master

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by sis0k0** [comment]
test

**Comment by sis0k0** [comment]
test

**Comment by sis0k0** [comment]
test

**Comment by Mitko-Kerezov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test branch_tns_core_modules#release

**Pull request merged**